### ISSUE TITLE
Monaco Bathymetry

### ISSUE THREAD

**Issue by jgwinner** [tilezen/joerd#190]
Hello:

I was using a program that uses the AWS Mapzen data, and combines with OpenStreetMap, but I'm getting some weird results around the Monaco shoreline. Is this a bug, or is it a result of the source data?

The source is the area: 7.39886, 43.70961, 7.46272, 43.75698

![image](https://user-images.githubusercontent.com/10425170/86519531-6c992d00-bdf0-11ea-89e9-c07b7d2d64ab.png)

**Comment by rmarianski** [comment]
IIRC, there were artifacts in the data around shorelines, but I'll defer to @nvkelso for the specifics.

**Comment by jgwinner** [comment]
Thanks. It's kind of a killer, we could hand edit the above, but we were trying to build a process that would take user inputs.

Do you know of any other data sets that could be used?

        == John ==

**Comment by nvkelso** [comment]
Yes, these are shoreline artifacts. Usually blending between different sources is tricky, but the super low resolution nature of the bathymetry here is the main problem. Do you know of a better open data source for Mediterranean Sea bathymetry that could be used here? That would probably fix things up.

**Comment by jgwinner** [comment]
I'm still exploring datasets. However, If I knew where the data sets were coming from, or turn on/off the bathymetry, I could build my own blending model in World Machine or Leveller, at least for my current application.

I'll explore your model better, I came here from another app that consumes your data. 

        == John ==
